HeartbeatLogging/Sources/Storage.swift

```diff
@@ -73,16 +96,27 @@ final class FileStorage: Storage {
 
 // MARK: - UserDefaultsStorage
 
+/// A object that provides API for reading and writing to a user defaults resource.
 final class UserDefaultsStorage: Storage {
+  /// The underlying defaults container.
   private let defaults: UserDefaults
+  /// The key mapping to the object's associated resource in `defaults`.
   private let key: String
 
+  /// Designated initializer.
+  /// - Parameters:
+  ///   - defaults: The defaults container. Defaults to the `standard` instance if `nil`.
+  ///   - key: The key mapping to the value stored in the defaults container.
   init(defaults: UserDefaults? = nil,
        key: String) {
     self.defaults = defaults ?? .standard
```

Comment:
I'm thinking maybe we should never use `.standard` by default since it just opens up room for errors?

Reply:
Good idea. I'm not sure how to maneuver around the fact that `UserDefaults`'s[`init?(suiteName suitename: String?)`](https://developer.apple.com/documentation/foundation/userdefaults/1409957-init) returns an optional instance. 

I guess it will be up to clients of this class to unwrap the defaults they pass in.

Reply:
Reworked in 29cf736